web/src/containers/ActivityDetailCostAllocate.js

```diff
@@ -28,8 +28,8 @@ const ActivityDetailCostAllocate = props => {
           reminder="activities.costAllocate.methodology.reminder"
         />
         <RichText
-          content={costAllocateDesc}
-          onSync={sync('costAllocateDesc')}
+          content={costAllocationDesc}
+          onSync={sync('costAllocationDesc')}
```

Comment:
Out of curiosity, do you have a tool or process for renaming things throughout an app, or do you just do it very carefully?  😛

Reply:
just `ack` and find/replace and patience :)